.github/workflows/scripts/ti_build/entry.py

```diff
@@ -34,38 +28,19 @@ def build_wheel(python: Command, pip: Command) -> None:
     """
 
     git.fetch("origin", "main", "--tags", "--force")
-    proj_tags = []
     extra = []
 
     cmake_args.writeback()
-    assert misc.options is not None
-    if misc.options.tag_local:
-        wheel_tag = f"+{misc.options.tag_local}"
-    elif misc.options.tag_config:
-        wheel_tag = f"+{cmake_args.render_wheel_tag()}"
-    else:
-        wheel_tag = ""
-
-    if misc.options.nightly:
-        os.environ["PROJECT_NAME"] = "gstaichi-nightly"
-        now = datetime.datetime.now().strftime("%Y%m%d")
-        proj_tags.extend(["egg_info", f"--tag-build=.post{now}{wheel_tag}"])
-    elif wheel_tag:
-        proj_tags.extend(["egg_info", f"--tag-build={wheel_tag}"])
-
     if platform.system() == "Linux":
-        if is_manylinux2014():
```

Comment:
removed the conditioanl choosing of manylinux 2014, since we dont support manylinux 2014